### ISSUE TITLE
Option to force entry of the commit message

### ISSUE THREAD

**Issue by gopf** [dkandalov/limited-wip#41]
I would like to have an option to force the entry of a new commit message before committing. I can use the commit dialog, but the old message stays there by default so if I forget to adapt it then the history is wrong / confusing. Maybe the message in the commit dialog can be cleared when it is opened (it won't allow an empty commit message to be used), or maybe there could be another option to enter a commit message through a separate dialog before committing.

**Comment by dkandalov** [comment]
> Maybe the message in the commit dialog can be cleared when it is opened
Yes, I think it's best option and it shouldn't be too hard to implement but it sounds a bit like an IDE feature 🤔

**Comment by dkandalov** [comment]
For example https://youtrack.jetbrains.com/issue/IDEA-216566
